### PR TITLE
In-2468 Add ubl/e-fff format to creditNotes.download

### PR DESCRIPTION
**Pull request by luisaterrabrito**

### CODE REVIEW

**Review by sanderbrugge — DISMISSED** [review]

**Review by EOSullivanBerlin — DISMISSED** [review]

**Review by lowiebenoot — DISMISSED** [review]

File: src/changes-backwards-compatible.apib [hunk 1]
```diff
@@ -13,6 +13,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 - We added the `weighted_value` to the response of both `deals.list` and `deals.info` requests.
 
+- We added format `ubl/e-fff` to `creditNotes.download`.
```
Comment (dietrichm) [inline src/changes-backwards-compatible.apib:18]: PP: add this to the top of the list, like we did in #486.

**Review by dietrichm — DISMISSED** [review]

**Comment by lowiebenoot** [comment]
Conflicts 🤷‍♂

**Review by lowiebenoot — APPROVED** [review]

**Review by EOSullivanBerlin — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]
PP still applies.

**Comment by luisaterrabrito** [comment]
I don't think the PP adds value.

**Comment by dietrichm** [comment]
> I don't think the PP adds value.

Okay, I do think it's weird that for one endpoint the new format is added at the top of the changelist while for the other it's at the bottom. But not blocking.

**Review by sanderbrugge — APPROVED** [review]

**Pull request merged**